src/shared/components/Terms/styles.scss

```diff
@@ -44,6 +44,7 @@
   font-size: 28px;
   margin-bottom: 1.5 * $base-unit;
   text-align: center;
+  color: $tc-gray-80;
```

Comment:
This as well related to another issue, please remove from this PR.

Reply:
Will remove this as well.